lib/cli/interactive-setup/console-setup-iam-role.js

```diff
@@ -100,10 +100,14 @@ module.exports = {
       await awsRequest(context, cloudFormationServiceConfig, 'describeStacks', {
         StackName: iamRoleStackName,
       });
-      log.warning(
+      const postMessage = isConsoleDevMode ? log.error : log.warning;
+      postMessage(
         'Cannot integrate with Serverless Console: ' +
-          'AWS account is already integrated with another org'
+          'AWS account is already integrated with another org. ' +
+          'You can use the --aws-profile option to use a different AWS Profile.'
```

Comment:
Is this intentional to show it either as an error or warning? I think we can just switch to error here, if that's the request

Reply:
It was but I guess this is true we should just change it to an error on both 👍